xclim/__init__.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import annotations
 
+import pathlib
```

Comment:
`From pathlib import Path`

As you just use this class and nothing other in the module

Reply:
Makes sense. I typically only ever import the class. Will modify, thanks!